python/tvm/relay/grammar/Relay.g4

```diff
@@ -20,7 +20,7 @@ NE: '!=' ;
 
 opIdent: CNAME ;
 GLOBAL_VAR: '@' CNAME ;
-LOCAL_VAR: '%' CNAME ;
+LOCAL_VAR: '%' (CNAME | INT);
```

Comment:
imo LOCAL_VAR and GRAPH_VAR should be separated at the lexing stage. This will simplify code paths in `_parser.py`.

Reply:
+1 to have a `GRAPH_VAR` definition. It would be clearer to write and read the parser code.

Reply:
For sure, I was just hacking to try and get errors working.